backend/routers/router_stock_food.py

```
@@ -0,0 +1,59 @@
+async def get_db():
+    async with AsyncSessionLocal() as session:
+        yield session
+
+
+@router.get("/", response_model=list[StockFoodOut])
+async def read_stock_food(db: AsyncSession = Depends(get_db)):
+    return await stock_food_service.get_foods(db)
+
+
+@router.get("/{food_id}", response_model=StockFoodOut)
+async def read_stock_food(
```

Ruff (0.14.10)

Redefinition of unused `read_stock_food` from line 16

(F811)